src/styles/blocks/footer.scss

```diff
@@ -0,0 +1,17 @@
+.footer {
+  position: sticky;
+  height: 200px;
+  z-index: -1;
+
+  @include on-tablet {
+    height: 480px;
+  }
+
+  @include on-desktop {
+    height: 555px;
+  }
+
+  background-image: url(../images/footer-img.png);
+  background-position: center;
+  background-size: cover;
+}
```

Comment:
There's the last point in checklist: Make the footer sticky at the bottom with z-index: -1 to get a fixed background effect.